File: cpp/Program/Anggota.cpp
#include <iostream>
#include <string>

using namespace std;

class Anggota{
    private:
        int id;
        string name;
        string bidang;
        string partai;

    public:
        Anggota(){
            this->name = "";
            this->bidang = '-';
        }
        
        Anggota(int id, string name, string bidang, string partai){
            this->id = id;
            this->name = name;
            this->bidang = bidang;
            this->partai = partai;
        }

        int getId(){
            return this->id;
        }

        void setId(int id){
            this->id = id;
        }

        string getName(){
            return this->name;
        }

        void setName(string name){
            this->name = name;
        }

        string getBidang(){
            return this->bidang;
        }

        void setBidang(string bidang){
            this->bidang = bidang;
        }

        string getPartai(){
            return this->partai;
        }

        void setPartai(string partai){
            this->partai = partai;
        }

        ~Anggota(){

        }
};

File: py/Program/Anggota.py
class Anggota:
    __id = ""
    __nama = ""
    __bidang = ""
    __partai = ""

    def __init__(self, id = "", nama = "", bidang = "", partai = ""):
        self.__id = id
        self.__nama = nama
        self.__bidang = bidang
        self.__partai = partai
    
    def getId(self):
        return self.__id
    
    def setId(self, id):
        self.__id = id
    
    def getNama(self):
        return self.__nama
    
    def setNama(self, nama):
        self.__nama = nama
    
    def getBidang(self):
        return self.__bidang
    
    def setBidang(self, bidang):
        self.__bidang = bidang
    
    def getPartai(self):
        return self.__partai
    
    def setPartai(self, partai):
        self.__partai = partai